azure-pipelines.yml

```diff
@@ -151,13 +151,13 @@ jobs:
       - script: |
           yarn workspace @fluentui/docs vr:build
         displayName: build FUI N* VR Test
-        condition: ne(variables['SKIP_SCREENER_BUILD'], 'true')
+        condition: eq(variables['IS_ARTIFACT_PRESENT'], 'true')
```

Comment:
Should be this inverted?

Reply:
before it was building if `SKIP_SCREENER_BUILD` was not equal to true (so it was false), now it is building when `IS_ARTIFACT_PRESENT` is true, so the screener build should not be skipped, hope it makes sense